seatunnel-connectors-v2/connector-file/connector-file-base/src/test/java/org/apache/seatunnel/connectors/seatunnel/file/writer/ParquetReadStrategyTest.java

```diff
@@ -192,4 +227,69 @@ public String getSchema() {
             return SCHEMA;
         }
     }
+
+    public static class AutoGenerateParquetData {
+
+        public static final String DATA_FILE_PATH = "/tmp/data.parquet";
+
+        public static void generateTestData() throws IOException {
+            System.out.println("generateTestData start.");
```

Comment:
Please use log instead of `System.out.println`.

Reply:
ok